Makefile

```diff
@@ -13,7 +13,7 @@ VERSION ?= $(shell git tag --points-at HEAD | grep ^v | head -n 1)
 LDFLAGS = -ldflags "-X main.BuildTime=$(BUILD_TIME) -X main.GitCommit=$(GIT_COMMIT) -X main.Version=$(VERSION)"
 
 .PHONY: all
-all: audit test build lint test-component ## Runs multiple targets, audit, lint, test and test-component
```

Comment:
It looks like we've lost the `build` task (ignore this comment if it wasn't actually needed).

Reply:
good point, I think we should keep it in.